metricbeat/module/system/process/_meta/fields.yml

```diff
@@ -5,38 +5,35 @@
   release: ga
   fields:
     - name: name
-      type: keyword
-      description: >
-        The process name.
+      type: alias
+      path: process.name
+      migration: true
     - name: state
       type: keyword
       description: >
         The process state. For example: "running".
     - name: pid
-      type: long
-      description: >
-        The process pid.
+      type: alias
+      path: process.pid
+      migration: true
     - name: ppid
-      type: long
-      description: >
-        The process parent pid.
+      type: alias
+      path: process.ppid
+      migration: true
     - name: pgid
-      type: long
-      description: >
-        The process group id.
+      type: alias
+      path: process.pgid
+      migration: true
     - name: cmdline
       type: keyword
       description: >
         The full command-line used to start the process, including the
         arguments separated by space.
       ignore_above: 2048
     - name: username
-      type: keyword
-      description: >
-        The username of the user that created the process. If the username
-        cannot be determined, the field will contain the user's
-        numeric identifier (UID). On Windows, this field includes the user's
-        domain and is formatted as `domain\username`.
+      type: alias
+      path: user.name
+      migration: true
     - name: cwd
```

Comment:
`cwd` should be migrated to `process.working_directory`

Reply:
Oh, good catch, thanks.

Reply:
Fixed